# Dimerización de $NO_2$

```{warning}
Si está utilizando Google Colab o la ejecución en línea, recuerde que debe de ejecutar al inicio el siguiente código
~~~python
!wget -c https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
!chmod +x Miniconda3-latest-Linux-x86_64.sh
!bash ./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local
!conda install -y psi4 python=3.7 -c psi4
import sys
sys.path.append("/usr/local/lib/python3.7/site-packages/")
~~~
```

El $N_2O_4$ es un compuesto que se encuentra en la atmósfera. Con los cambios de temperatura se descompone en el radical $NO_2$ mediante la reacción:

$N_2O_4 <=> 2NO_2$

Las geometrías de $N_2O_4$ y $NO_2$ se dan a continuación:

Molécula: $N_2O_4$ Carga: 0 Multiplicidad: 1

|Átomo |x (Å)  |y (Å)  |z (Å) |
|------|-------|-------|------|
|N     |0.0000 |0.0000 |0.0000|
|N     |-1.7820|0.0000 |0.0000|
|O     |0.4516 |1.1010 |0.0000|
|O     |0.4516 |-1.1010|0.0000|
|O     |-2.2336|1.1010 |0.0000|
|O     |-2.2336|-1.1010|0.0000|

Molécula: $NO_2$ Carga: 0 Multiplicidad: 2

|Átomo |x (Å)  |y (Å)  |z (Å) |
|------|-------|-------|------|
|N     | 0.0000| 0.0000|0.0000|
|O     | 0.0000| 1.0989|0.4653|
|O     | 0.0000|-1.0989|0.4653|


In [1]:
# Importe Psi4

In [2]:
# Descomentar estas líneas si está en modo online

#!wget -c https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
#!chmod +x Miniconda3-latest-Linux-x86_64.sh
#!bash ./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local
#!conda install -y psi4 python=3.7 -c psi4
#import sys
#sys.path.append("/usr/local/lib/python3.7/site-packages/")

import psi4

**Pregunta 1.** Calcule la energía de la molécula de $N_2O_4$ con HF y la base aug-cc-pvdz. 

In [3]:
psi4.set_memory("2 gb")
psi4.geometry("""
0 1
   N       -4.84638        1.76109        0.00000
   N       -3.46888        1.78415        0.00000
   O       -2.82385        2.93169       -0.00000
   O       -2.85055        0.76276        0.00000
   O       -5.46471        2.78248        0.00000
   O       -5.49141        0.61355        0.00000
""")
n2o4,wfn=psi4.opt("HF/aug-cc-pvdz", return_wfn=True)
print(n2o4)


  Memory set to   1.863 GiB by Python driver.

Scratch directory: /tmp/
gradient() will perform analytic gradient computation.

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:41 2022

   => Loading Basis Set <=

    Name: AUG-CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry N          line   218 file /home/jfhlewyee/anaconda3/envs/jb/share/psi4/basis/aug-cc-pvdz.gbs 
    atoms 3-6 entry O          line   254 file /home/jfhlewyee/anaconda3/envs/jb/share/psi4/basis/aug-cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,   1907 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in An

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 1.397 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0105
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.1174353001E-04.
  Reciprocal condition number of the overlap matrix is 1.5709236004E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly at

   @DF-RHF iter   1:  -407.60427833466974   -3.68225e-01   1.40862e-02 ADIIS/DIIS
   @DF-RHF iter   2:  -407.54559070008878    5.86876e-02   1.60795e-02 ADIIS/DIIS
   @DF-RHF iter   3:  -408.02844149265133   -4.82851e-01   9.54659e-04 ADIIS/DIIS
   @DF-RHF iter   4:  -408.03188024993545   -3.43876e-03   5.80492e-04 ADIIS/DIIS
   @DF-RHF iter   5:  -408.03301304803875   -1.13280e-03   2.76638e-04 ADIIS/DIIS


   @DF-RHF iter   6:  -408.03338559967676   -3.72552e-04   1.06189e-04 ADIIS/DIIS
   @DF-RHF iter   7:  -408.03350898881388   -1.23389e-04   5.90493e-05 DIIS
   @DF-RHF iter   8:  -408.03354808515189   -3.90963e-05   2.09052e-05 DIIS
   @DF-RHF iter   9:  -408.03355353420858   -5.44906e-06   4.50819e-06 DIIS
   @DF-RHF iter  10:  -408.03355371410271   -1.79894e-07   1.44494e-06 DIIS
   @DF-RHF iter  11:  -408.03355372865087   -1.45482e-08   4.53581e-07 DIIS
   @DF-RHF iter  12:  -408.03355373008230   -1.43143e-09   1.03362e-07 DIIS
   @DF-RHF iter  13:  -408.03355373015035   -6.80416e-11   2.43198e-08 DIIS
   @DF-RHF iter  14:  -408.03355373015262   -2.27374e-12   7.46914e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag   -20.734126     1Bu   -20.734124     2Ag   -20.677034  
       2Bu   -20.677031     3Ag   -15.967262  


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:43 2022
Module time:
	user time   =       1.40 seconds =       0.02 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =       1.40 seconds =       0.02 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:43 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 1.397 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.0934102566E-04.
  Reciprocal condition number of the overlap matrix is 1.5127240688E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Orbitals guess was supplied from a previous computa

   @DF-RHF iter   4:  -408.09720470230963   -6.15333e-04   1.24730e-04 ADIIS/DIIS
   @DF-RHF iter   5:  -408.09734560748450   -1.40905e-04   4.41401e-05 DIIS
   @DF-RHF iter   6:  -408.09736957335622   -2.39659e-05   2.37826e-05 DIIS
   @DF-RHF iter   7:  -408.09737749909578   -7.92574e-06   7.18943e-06 DIIS


   @DF-RHF iter   8:  -408.09737790089855   -4.01803e-07   5.57288e-06 DIIS
   @DF-RHF iter   9:  -408.09737806421606   -1.63318e-07   7.95771e-07 DIIS
   @DF-RHF iter  10:  -408.09737806966950   -5.45344e-09   2.04890e-07 DIIS
   @DF-RHF iter  11:  -408.09737806998078   -3.11275e-10   4.72209e-08 DIIS
   @DF-RHF iter  12:  -408.09737806999516   -1.43814e-11   1.12330e-08 DIIS
   @DF-RHF iter  13:  -408.09737806999556   -3.97904e-13   2.79119e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag   -20.703743     1Bu   -20.703742     2Ag   -20.679692  
       2Bu   -20.679691     3Ag   -15.956685     3Bu   -15.956216  
       4Ag    -1.775387     4Bu    -1.682042     5Bu    -1.527764  
       5Ag    -1.484202     6Ag    -1.196570     6Bu    -0.903944  
       7Ag    -0.886740     1Au    -0.867487     7Bu    -0.830656  
       8


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:45 2022
Module time:
	user time   =       1.24 seconds =       0.02 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       3.73 seconds =       0.06 minutes
	system time =       0.12 seconds =       0.00 minutes
	total time  =          4 seconds =       0.07 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:45 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 1.397 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.1530600313E-04.
  Reciprocal condition number of the overlap matrix is 1.5789590135E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Orbitals guess was supplied from a previous computa

   @DF-RHF iter   4:  -408.10300298057729   -3.43894e-04   9.59315e-05 DIIS
   @DF-RHF iter   5:  -408.10309782775016   -9.48472e-05   4.32239e-05 DIIS
   @DF-RHF iter   6:  -408.10311388959383   -1.60618e-05   1.77948e-05 DIIS
   @DF-RHF iter   7:  -408.10311683380701   -2.94421e-06   1.12911e-05 DIIS
   @DF-RHF iter   8:  -408.10311757777555   -7.43969e-07   2.18719e-06 DIIS
   @DF-RHF iter   9:  -408.10311762465909   -4.68835e-08   3.76334e-07 DIIS
   @DF-RHF iter  10:  -408.10311762578829   -1.12919e-09   1.20170e-07 DIIS
   @DF-RHF iter  11:  -408.10311762589276   -1.04478e-10   3.53130e-08 DIIS
   @DF-RHF iter  12:  -408.10311762590004   -7.27596e-12   6.86537e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag   -20.686404     1Bu   -20.686403     2Ag   -20.672568  
       2Bu   -20.672566     3Ag   -15.940821     3Bu


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:48 2022
Module time:
	user time   =       1.21 seconds =       0.02 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =       6.03 seconds =       0.10 minutes
	system time =       0.18 seconds =       0.00 minutes
	total time  =          7 seconds =       0.12 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:48 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 1.397 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.3327196856E-04.
  Reciprocal condition number of the overlap matrix is 1.8550413836E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Orbitals guess was supplied from a previous computa

   @DF-RHF iter   5:  -408.11150317606558   -4.85523e-05   2.62176e-05 DIIS
   @DF-RHF iter   6:  -408.11151251637244   -9.34031e-06   2.05790e-05 DIIS
   @DF-RHF iter   7:  -408.11151508380010   -2.56743e-06   5.58197e-06 DIIS
   @DF-RHF iter   8:  -408.11151544050995   -3.56710e-07   2.15188e-06 DIIS
   @DF-RHF iter   9:  -408.11151547906479   -3.85548e-08   2.77955e-07 DIIS
   @DF-RHF iter  10:  -408.11151547948066   -4.15866e-10   4.49772e-08 DIIS
   @DF-RHF iter  11:  -408.11151547949180   -1.11413e-11   1.30081e-08 DIIS
   @DF-RHF iter  12:  -408.11151547949311   -1.30740e-12   4.27077e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag   -20.696523     1Bu   -20.696522     2Ag   -20.687181  
       2Bu   -20.687180     3Ag   -15.950257     3Bu   -15.949954  
       4Ag    -1.756456     4Bu    -1.685431     5Bu    -1.5


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:50 2022
Module time:
	user time   =       1.18 seconds =       0.02 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       8.30 seconds =       0.14 minutes
	system time =       0.28 seconds =       0.00 minutes
	total time  =          9 seconds =       0.15 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:50 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.3253257352E-04.
  Reciprocal condition number of the overlap matrix is 1.8398659033E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Orbitals guess was supplied from a previous computation.

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta  


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:52 2022
Module time:
	user time   =       1.18 seconds =       0.02 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =      10.56 seconds =       0.18 minutes
	system time =       0.35 seconds =       0.01 minutes
	total time  =         11 seconds =       0.18 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:52 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  Reading orbitals from file /tmp/stdout.default.554391.180.npy, no projection.

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 1.397 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.3671256039E-04.
  Reciprocal condition number of the overlap matrix is 1.8989434941E-05.
    Using symmetric orthogonalization.

  ==> Pre

   @DF-RHF iter   8:  -408.11510677637909   -4.34210e-09   1.91710e-07 DIIS
   @DF-RHF iter   9:  -408.11510677662835   -2.49258e-10   8.20347e-08 DIIS
   @DF-RHF iter  10:  -408.11510677666058   -3.22302e-11   3.00405e-08 DIIS
   @DF-RHF iter  11:  -408.11510677666615   -5.57066e-12   5.83904e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag   -20.688764     1Bu   -20.688761     2Bu   -20.688523  
       2Ag   -20.688523     3Ag   -15.944959     3Bu   -15.944698  
       4Ag    -1.763475     4Bu    -1.700570     5Bu    -1.547547  
       5Ag    -1.514371     6Ag    -1.112818     6Bu    -0.901529  
       7Ag    -0.887088     1Au    -0.852702     7Bu    -0.822286  
       8Ag    -0.796744     1Bg    -0.776189     8Bu    -0.691221  
       9Bu    -0.609865     2Bg    -0.539644     9Ag    -0.538299  
       2Au    -0.519775 


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:55 2022
Module time:
	user time   =       1.16 seconds =       0.02 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          2 seconds =       0.03 minutes
Total time:
	user time   =      12.80 seconds =       0.21 minutes
	system time =       0.44 seconds =       0.01 minutes
	total time  =         14 seconds =       0.23 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:55 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

 Ndocc  Nsocc
   -------------------------------------------------------
     Ag        48      48      10      10      10       0
     Bg        21      21       2       2       2       0
     Au        21      21       2       2       2       0
     Bu        48      48       9       9       9       0
   -------------------------------------------------------
    Total     138     138      23      23      23       0
   -------------------------------------------------------

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   0:  -408.12165131063466   -4.08122e+02   4.53095e-04 
   @DF-RHF iter   1:  -408.11516230588961    6.48900e-03   4.76775e-05 DIIS
   @DF-RHF iter   2:  -408.11516969854426   -7.39265e-06   1.67599e-05 DIIS
   @DF-RHF iter   3:  -408.11517082291761   -1.12437e-06   9.09450e-06 DIIS
   @DF-RHF iter   4:  -408.11517118684333   -3.63926e-07   4.11005e-06 DIIS
   @DF-RHF iter   5:  -408.11517129090686   -1.0

33914  
      12Ag     0.143241    12Bu     0.177878     4Bg     0.190170  
      13Ag     0.192160    14Ag     0.203650    13Bu     0.211767  
       5Bg     0.214698    14Bu     0.260492    15Bu     0.262222  
       5Au     0.269643    16Bu     0.274933     6Au     0.283855  
      15Ag     0.322176    16Ag     0.322257    17Ag     0.335159  
       6Bg     0.348854    18Ag     0.353580    17Bu     0.373152  
      18Bu     0.410784    19Bu     0.448992    19Ag     0.468526  
      20Ag     0.538579     7Au     0.592636    20Bu     0.598058  
      21Bu     0.630855     7Bg     0.708247    21Ag     0.716708  
      22Bu     0.795848    23Bu     0.808624     8Bg     0.814176  
      22Ag     0.814519    23Ag     0.860306    24Bu     0.893184  
       8Au     0.909205    24Ag     0.943036    25Bu     0.963717  
      25Ag     1.057328     9Bg     1.077220     9Au     1.093576  
      26Ag     1.102790    10Bg     1.106816    10Au     1.167504  
      11Bg     1.167863    11Au     1.17


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:57 2022
Module time:
	user time   =       1.18 seconds =       0.02 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =      15.06 seconds =       0.25 minutes
	system time =       0.50 seconds =       0.01 minutes
	total time  =         16 seconds =       0.27 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:57 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:              1430
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.3921580759E-04.
  Reciprocal condition number of the overlap matrix is 1.9349557366E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Orbitals guess was supplied from a previous computation.

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta  


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:51:59 2022
Optimizer: Optimization complete!
-408.1151750819947


Module time:
	user time   =       1.05 seconds =       0.02 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =      17.20 seconds =       0.29 minutes
	system time =       0.58 seconds =       0.01 minutes
	total time  =         18 seconds =       0.30 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:51:59 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2h
    Full point group: C2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z            

**Pregunta 2.** Calcule la energía de la molécula de $NO_2$ con HF y la base aug-cc-pvdz. [Complete donde haga falta - Reemplace las X]

In [4]:
NO2 = psi4.geometry("""
0 2
   N       -4.39539        1.87380        0.00000
   O       -3.90978        3.09520       -0.00000
   O       -3.65594        0.93810        0.00000
units angstrom
""")
psi4.set_options({'reference': 'uhf'})
no2=psi4.optimize("HF/aug-cc-pvdz")
print(no2)

	Removing binary optimization data file.
	Cleaning optimization helper files.

Scratch directory: /tmp/
gradient() will perform analytic gradient computation.

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:00 2022

   => Loading Basis Set <=

    Name: AUG-CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry N          line   218 file /home/jfhlewyee/anaconda3/envs/jb/share/psi4/basis/aug-cc-pvdz.gbs 
    atoms 2-3 entry O          line   254 file /home/jfhlewyee/anaconda3/envs/jb/share/psi4/basis/aug-cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              UHF Reference
                        1 Threads,   1907 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point g

ormation).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A'        48      48 
     A"        21      21 
   -------------------------
    Total      69      69
   -------------------------

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-UHF iter SAD:  -203.13738478568362   -2.03137e+02   0.00000e+00 
   @DF-UHF iter   1:  -203.88769942201003   -7.50315e-01   1.09934e-02 ADIIS/DIIS


   @DF-UHF iter   2:  -203.91435859319847   -2.66592e-02   1.05946e-02 ADIIS/DIIS
   @DF-UHF iter   3:  -204.02595177891368   -1.11593e-01   1.27232e-03 ADIIS/DIIS
   @DF-UHF iter   4:  -204.02902305136723   -3.07127e-03   8.50272e-04 ADIIS/DIIS
   @DF-UHF iter   5:  -204.03036450178428   -1.34145e-03   3.57192e-04 ADIIS/DIIS
   @DF-UHF iter   6:  -204.03093217267732   -5.67671e-04   1.82155e-04 ADIIS/DIIS
   @DF-UHF iter   7:  -204.03122863250778   -2.96460e-04   1.29823e-04 ADIIS/DIIS
   @DF-UHF iter   8:  -204.03143898072892   -2.10348e-04   1.22488e-04 ADIIS/DIIS
   @DF-UHF iter   9:  -204.03153633637524   -9.73556e-05   1.26291e-04 ADIIS/DIIS
   @DF-UHF iter  10:  -204.03113553601506    4.00800e-04   1.02446e-04 ADIIS/DIIS
   @DF-UHF iter  11:  -204.03084627848352    2.89258e-04   7.63819e-05 DIIS
   @DF-UHF iter  12:  -204.03082122256211    2.50559e-05   5.18759e-05 DIIS
   @DF-UHF iter  13:  -204.03083249407959   -1.12715e-05   4.29948e-05 DIIS
   @DF-UHF iter  14:  -204.0308516

   @DF-UHF iter  24:  -204.03113206954112   -7.89838e-06   2.85041e-05 DIIS
   @DF-UHF iter  25:  -204.03113387710405   -1.80756e-06   8.04664e-06 DIIS
   @DF-UHF iter  26:  -204.03113486467453   -9.87570e-07   5.20647e-06 DIIS
   @DF-UHF iter  27:  -204.03113455175608    3.12918e-07   5.78488e-06 DIIS


   @DF-UHF iter  28:  -204.03113467836840   -1.26612e-07   3.22147e-06 DIIS
   @DF-UHF iter  29:  -204.03113376380003    9.14568e-07   8.59048e-06 DIIS
   @DF-UHF iter  30:  -204.03113452918976   -7.65390e-07   2.39648e-06 DIIS
   @DF-UHF iter  31:  -204.03113445561689    7.35729e-08   2.70400e-06 DIIS
   @DF-UHF iter  32:  -204.03113455466917   -9.90523e-08   6.12645e-07 DIIS
   @DF-UHF iter  33:  -204.03113455586379   -1.19462e-09   2.57614e-07 DIIS
   @DF-UHF iter  34:  -204.03113455733444   -1.47065e-09   8.11414e-08 DIIS
   @DF-UHF iter  35:  -204.03113455743829   -1.03853e-10   3.50612e-08 DIIS
   @DF-UHF iter  36:  -204.03113455744750   -9.20863e-12   1.54606e-08 DIIS
   @DF-UHF iter  37:  -204.03113455744668    8.24230e-13   9.28692e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contamination Metric:   1.067674469E-01
   @S^2 Expected:                7.500000000E-01
   @S^2 Observed:                8.567674469E-01
   @S   Expected:         


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:01 2022
Module time:
	user time   =       0.52 seconds =       0.01 minutes
	system time =       0.00 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =      18.81 seconds =       0.31 minutes
	system time =       0.63 seconds =       0.01 minutes
	total time  =         20 seconds =       0.33 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:01 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 2:

       Cen

   1.150910
	    3 B(2,1,3)        =    119.999940     0.006412    16.182273   136.182212
	 ---------------------------------------------------------------------------
	Successfully symmetrized geometry.

  ==> Convergence Check <==

  Measures of convergence in internal coordinates in au.
  Criteria marked as inactive (o), active & met (*), and active & unmet ( ).
  --------------------------------------------------------------------------------------------- ~
   Step     Total Energy     Delta E     MAX Force     RMS Force      MAX Disp      RMS Disp    ~
  --------------------------------------------------------------------------------------------- ~
    Convergence Criteria    1.00e-06 *    3.00e-04 *             o    1.20e-03 *             o  ~
  --------------------------------------------------------------------------------------------- ~
      1    -204.03113456   -2.04e+02      1.09e-01      9.49e-02 o    2.82e-01      1.93e-01 o  ~
  ------------------------------------------

   @DF-UHF iter  10:  -204.06340741208712   -2.47711e-05   3.54556e-05 DIIS
   @DF-UHF iter  11:  -204.06342955693441   -2.21448e-05   3.24747e-05 DIIS
   @DF-UHF iter  12:  -204.06346143301343   -3.18761e-05   2.99245e-05 DIIS
   @DF-UHF iter  13:  -204.06350260688134   -4.11739e-05   2.74382e-05 DIIS
   @DF-UHF iter  14:  -204.06355069495942   -4.80881e-05   2.49029e-05 DIIS
   @DF-UHF iter  15:  -204.06358018914833   -2.94942e-05   2.22117e-05 DIIS
   @DF-UHF iter  16:  -204.06361690772562   -3.67186e-05   2.19287e-05 DIIS
   @DF-UHF iter  17:  -204.06365662726023   -3.97195e-05   1.30705e-05 DIIS
   @DF-UHF iter  18:  -204.06366880755229   -1.21803e-05   6.50620e-06 DIIS
   @DF-UHF iter  19:  -204.06367031149151   -1.50394e-06   2.33243e-06 DIIS
   @DF-UHF iter  20:  -204.06367036830471   -5.68132e-08   4.59310e-06 DIIS
   @DF-UHF iter  21:  -204.06367041495591   -4.66512e-08   1.47192e-06 DIIS
   @DF-UHF iter  22:  -204.06367042591057   -1.09547e-08   3.76400e-07 DIIS
   @DF-UHF i


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:01 2022
Module time:
	user time   =       0.38 seconds =       0.01 minutes
	system time =       0.00 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =      19.36 seconds =       0.32 minutes
	system time =       0.64 seconds =       0.01 minutes
	total time  =         20 seconds =       0.33 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:01 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 2:

       Cen

   @DF-UHF iter  11:  -204.06335214363975   -1.64417e-06   4.92704e-06 DIIS
   @DF-UHF iter  12:  -204.06335245676951   -3.13130e-07   3.11797e-06 DIIS
   @DF-UHF iter  13:  -204.06335261704461   -1.60275e-07   1.57491e-06 DIIS
   @DF-UHF iter  14:  -204.06335265906196   -4.20173e-08   6.78641e-07 DIIS
   @DF-UHF iter  15:  -204.06335266347907   -4.41710e-09   3.37418e-07 DIIS
   @DF-UHF iter  16:  -204.06335266403681   -5.57748e-10   1.52500e-07 DIIS
   @DF-UHF iter  17:  -204.06335266414601   -1.09196e-10   8.43885e-08 DIIS
   @DF-UHF iter  18:  -204.06335266416545   -1.94404e-11   2.85225e-08 DIIS
   @DF-UHF iter  19:  -204.06335266416914   -3.69482e-12   9.81639e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contamination Metric:   1.417962189E-02
   @S^2 Expected:                7.500000000E-01
   @S^2 Observed:                7.641796219E-01
   @S   Expected:                5.000000000E-01
   @S   Observed:                5.000000000E-01

   

    ------------   -----------------  -----------------  -----------------  -----------------
         N           -0.278371469339    -0.029143447102     0.000000000000    14.003074004430
         O           -0.008009678204     1.054251171076     0.000000000000    15.994914619570
         O            0.251715654671    -1.028736946323     0.000000000000    15.994914619570

  Nuclear repulsion =   68.864140187560508

  ==> Basis Set <==

  Basis Set: AUG-CC-PVDZ
    Blend: AUG-CC-PVDZ
    Number of shells: 27
    Number of basis functions: 69
    Number of Cartesian functions: 75
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> DFJKGrad: Density-Fitted SCF Gradients <==

    Gradient:                    1
    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory [MiB]:             1430
    Schwarz Cutoff:          1E-12
    Fitting Condition: 

   @DF-UHF iter  14:  -204.06707805215208   -3.17496e-06   4.15514e-06 DIIS
   @DF-UHF iter  15:  -204.06707908602522   -1.03387e-06   3.26424e-06 DIIS
   @DF-UHF iter  16:  -204.06707991724102   -8.31216e-07   2.41738e-06 DIIS
   @DF-UHF iter  17:  -204.06708013833747   -2.21096e-07   1.73481e-06 DIIS
   @DF-UHF iter  18:  -204.06708026757522   -1.29238e-07   1.07421e-06 DIIS
   @DF-UHF iter  19:  -204.06708030026053   -3.26853e-08   5.83403e-07 DIIS
   @DF-UHF iter  20:  -204.06708030904062   -8.78009e-09   2.70555e-07 DIIS
   @DF-UHF iter  21:  -204.06708031006508   -1.02446e-09   1.05624e-07 DIIS
   @DF-UHF iter  22:  -204.06708031014068   -7.56017e-11   4.81688e-08 DIIS
   @DF-UHF iter  23:  -204.06708031015995   -1.92699e-11   2.73600e-08 DIIS
   @DF-UHF iter  24:  -204.06708031017152   -1.15676e-11   1.52766e-08 DIIS
   @DF-UHF iter  25:  -204.06708031017780   -6.28120e-12   9.49130e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contaminatio


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:02 2022
Module time:
	user time   =       0.34 seconds =       0.01 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =      20.35 seconds =       0.34 minutes
	system time =       0.67 seconds =       0.01 minutes
	total time  =         21 seconds =       0.35 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:02 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 2:

       Cen

   @DF-UHF iter  11:  -204.06901831691897   -5.33850e-06   5.23615e-06 DIIS
   @DF-UHF iter  12:  -204.06901905647749   -7.39559e-07   3.48479e-06 DIIS
   @DF-UHF iter  13:  -204.06901932396809   -2.67491e-07   2.28593e-06 DIIS
   @DF-UHF iter  14:  -204.06901945443431   -1.30466e-07   1.37956e-06 DIIS
   @DF-UHF iter  15:  -204.06901948212385   -2.76895e-08   6.22301e-07 DIIS
   @DF-UHF iter  16:  -204.06901948509014   -2.96629e-09   2.75574e-07 DIIS
   @DF-UHF iter  17:  -204.06901948569461   -6.04473e-10   1.41179e-07 DIIS
   @DF-UHF iter  18:  -204.06901948577618   -8.15703e-11   5.09880e-08 DIIS
   @DF-UHF iter  19:  -204.06901948579161   -1.54330e-11   2.08886e-08 DIIS
   @DF-UHF iter  20:  -204.06901948579343   -1.81899e-12   8.93444e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contamination Metric:   1.666202811E-02
   @S^2 Expected:                7.500000000E-01
   @S^2 Observed:                7.666620281E-01
   @S   Expected:         

    ------------   -----------------  -----------------  -----------------  -----------------
         N           -0.296867135325    -0.026713627324     0.000000000000    14.003074004430
         O           -0.003786463665     1.076083858021     0.000000000000    15.994914619570
         O            0.263684847890    -1.052696868515     0.000000000000    15.994914619570

  Nuclear repulsion =   67.102306050352382

  ==> Basis Set <==

  Basis Set: AUG-CC-PVDZ
    Blend: AUG-CC-PVDZ
    Number of shells: 27
    Number of basis functions: 69
    Number of Cartesian functions: 75
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> DFJKGrad: Density-Fitted SCF Gradients <==

    Gradient:                    1
    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory [MiB]:             1430
    Schwarz Cutoff:          1E-12
    Fitting Condition: 

   @DF-UHF iter   9:  -204.06964421255790   -4.34117e-08   1.92160e-06 DIIS
   @DF-UHF iter  10:  -204.06964432425630   -1.11698e-07   1.61641e-06 DIIS
   @DF-UHF iter  11:  -204.06964451661230   -1.92356e-07   9.13561e-07 DIIS
   @DF-UHF iter  12:  -204.06964456815203   -5.15397e-08   4.88762e-07 DIIS
   @DF-UHF iter  13:  -204.06964457437925   -6.22722e-09   3.40751e-07 DIIS
   @DF-UHF iter  14:  -204.06964457721449   -2.83524e-09   2.14259e-07 DIIS
   @DF-UHF iter  15:  -204.06964457824418   -1.02969e-09   1.15725e-07 DIIS
   @DF-UHF iter  16:  -204.06964457838080   -1.36623e-10   6.04834e-08 DIIS
   @DF-UHF iter  17:  -204.06964457842128   -4.04725e-11   3.56403e-08 DIIS
   @DF-UHF iter  18:  -204.06964457842884   -7.56017e-12   2.38592e-08 DIIS
   @DF-UHF iter  19:  -204.06964457843441   -5.57066e-12   1.24700e-08 DIIS
   @DF-UHF iter  20:  -204.06964457843583   -1.42109e-12   4.57503e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contaminatio

	user time   =       0.31 seconds =       0.01 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =      21.32 seconds =       0.36 minutes
	system time =       0.68 seconds =       0.01 minutes
	total time  =         22 seconds =       0.37 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:03 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 2:

       Center              X                  Y                   Z               Mass       


   @DF-UHF iter   8:  -204.06967643949605   -7.94918e-09   8.81794e-07 DIIS
   @DF-UHF iter   9:  -204.06967644804422   -8.54817e-09   8.16658e-07 DIIS
   @DF-UHF iter  10:  -204.06967647169088   -2.36467e-08   6.70910e-07 DIIS
   @DF-UHF iter  11:  -204.06967650557397   -3.38831e-08   3.95258e-07 DIIS
   @DF-UHF iter  12:  -204.06967651444978   -8.87582e-09   2.47397e-07 DIIS
   @DF-UHF iter  13:  -204.06967651623910   -1.78932e-09   1.74930e-07 DIIS
   @DF-UHF iter  14:  -204.06967651720149   -9.62388e-10   9.88291e-08 DIIS
   @DF-UHF iter  15:  -204.06967651743278   -2.31296e-10   5.32356e-08 DIIS
   @DF-UHF iter  16:  -204.06967651745606   -2.32774e-11   2.62021e-08 DIIS
   @DF-UHF iter  17:  -204.06967651746226   -6.19593e-12   1.60619e-08 DIIS
   @DF-UHF iter  18:  -204.06967651746393   -1.67688e-12   1.04224e-08 DIIS
   @DF-UHF iter  19:  -204.06967651746476   -8.24230e-13   5.06005e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contaminatio


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:04 2022
Module time:
	user time   =       0.29 seconds =       0.00 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =      21.78 seconds =       0.36 minutes
	system time =       0.69 seconds =       0.01 minutes
	total time  =         23 seconds =       0.38 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:04 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 2:

       Cen

   @DF-UHF iter   9:  -204.06967820916211   -2.26606e-10   1.26942e-07 DIIS
   @DF-UHF iter  10:  -204.06967820965787   -4.95760e-10   1.06427e-07 DIIS
   @DF-UHF iter  11:  -204.06967821035184   -6.93973e-10   7.30438e-08 DIIS
   @DF-UHF iter  12:  -204.06967821067192   -3.20085e-10   4.68941e-08 DIIS
   @DF-UHF iter  13:  -204.06967821076526   -9.33369e-11   2.93047e-08 DIIS
   @DF-UHF iter  14:  -204.06967821079405   -2.87912e-11   1.73506e-08 DIIS
   @DF-UHF iter  15:  -204.06967821079877   -4.71800e-12   1.21934e-08 DIIS
   @DF-UHF iter  16:  -204.06967821080067   -1.90425e-12   8.30138e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   @Spin Contamination Metric:   1.693376049E-02
   @S^2 Expected:                7.500000000E-01
   @S^2 Observed:                7.669337605E-01
   @S   Expected:                5.000000000E-01
   @S   Observed:                5.000000000E-01

    Orbital Energies [Eh]
    ---------------------

    Alpha Occupied:       

	user time   =       0.28 seconds =       0.00 minutes
	system time =       0.00 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =      22.23 seconds =       0.37 minutes
	system time =       0.70 seconds =       0.01 minutes
	total time  =         23 seconds =       0.38 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:04 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: cs
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 2:

       Center              X                  Y                   Z               Mass       


**Pregunta a.** Calcule el $\Delta U$ de la reacción $N_2O_4 <=> 2NO_2$ según HF.

In [5]:
(2*no2-n2o4)*2625.5

-63.48810713720107

**Pregunta 3.** Calcule la energía de la molécula de $N_2O_4$ con DFT B3LYP y la base aug-cc-pvdz.

In [6]:
psi4.set_memory("2 gb")
psi4.geometry("""
0 1
N     0.0000  0.0000  0.0000
N     -1.7820 0.0000  0.0000
O     0.4516  1.1010  0.0000
O     0.4516  -1.1010 0.0000
O     -2.2336 1.1010  0.0000
O     -2.2336 -1.1010 0.0000
""")
n2o4,wfn=psi4.opt("B3LYP/aug-cc-pvdz", return_wfn=True)
print(n2o4)

	Removing binary optimization data file.
	Cleaning optimization helper files.

  Memory set to   1.863 GiB by Python driver.

Scratch directory: /tmp/
gradient() will perform analytic gradient computation.

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:04 2022

   => Loading Basis Set <=

    Name: AUG-CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry N          line   218 file /home/jfhlewyee/anaconda3/envs/jb/share/psi4/basis/aug-cc-pvdz.gbs 
    atoms 3-6 entry O          line   254 file /home/jfhlewyee/anaconda3/envs/jb/share/psi4/basis/aug-cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              UKS Reference
                        1 Threads,   1907 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==


  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 0.927 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:               949
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0210
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Cached 100.0% of DFT collocation blocks in 0.469 [GiB].

  Minimum eigenvalue in the overlap matrix is 2.0142268335E-04.
  Reciprocal condition number of the overlap matrix is 2.9966340456E-05.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF

tomic Densities via on-the-fly atomic UHF (no occupation information).



   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     Ag        27      27 
     B1g       21      21 
     B2g       12      12 
     B3g        9       9 
     Au         9       9 
     B1u       12      12 
     B2u       21      21 
     B3u       27      27 
   -------------------------
    Total     138     138
   -------------------------

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-UKS iter SAD:  -409.98189407085431   -4.09982e+02   0.00000e+00 
   @DF-UKS iter   1:  -410.01165405274241   -2.97600e-02   1.58098e-02 ADIIS/DIIS
   @DF-UKS iter   2:  -409.28525316106641    7.26401e-01   3.18142e-02 ADIIS/DIIS
   @DF-UKS iter   3:  -410.23602930608433   -9.50776e-01   3.03261e-03 ADIIS/DIIS
   @DF-UKS iter   4:  -410.24592922006246   -9.89991e-03   5.26890e-04 ADIIS/DIIS
   @DF-UKS iter   5:  -410.24625473143107   -3.25511e-04   4.28021e-05 DIIS


   @DF-UKS iter   6:  -410.24625648373177   -1.75230e-06   5.25168e-06 DIIS
   @DF-UKS iter   7:  -410.24625651812505   -3.43933e-08   1.33725e-06 DIIS
   @DF-UKS iter   8:  -410.24625652056318   -2.43813e-09   1.02845e-07 DIIS
   @DF-UKS iter   9:  -410.24625652058495   -2.17710e-11   2.45341e-08 DIIS
   @DF-UKS iter  10:  -410.24625652058597   -1.02318e-12   1.23264e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   Electrons on quadrature grid:
      Nalpha   =   22.9999980572 ; deviation = -1.943e-06
      Nbeta    =   22.9999980572 ; deviation = -1.943e-06
      Ntotal   =   45.9999961145 ; deviation = -3.886e-06 

   @Spin Contamination Metric:  -2.486899575E-14
   @S^2 Expected:                0.000000000E+00
   @S^2 Observed:               -2.486899575E-14
   @S   Expected:                0.000000000E+00
   @S   Observed:                0.000000000E+00

    Orbital Energies [Eh]
    ---------------------

    Alpha Occupied:                          


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:13 2022
Module time:
	user time   =       8.26 seconds =       0.14 minutes
	system time =       0.10 seconds =       0.00 minutes
	total time  =          9 seconds =       0.15 minutes
Total time:
	user time   =      30.67 seconds =       0.51 minutes
	system time =       0.81 seconds =       0.01 minutes
	total time  =         32 seconds =       0.53 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:13 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: d2h
    Full point group: D2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  Reading orbitals from file /tmp/stdout.default.554391.180.npy, no projection.

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 0.928 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:               950
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0210
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Cached 100.0% of DFT collocation blocks in 0.469 [GiB].

  Minimum eigenvalue in the overlap matrix is 2.0459205918E-04.
  Reciprocal condition number of the overlap matrix is 3.053922

   @DF-UKS iter   5:  -410.24642384356667   -7.84991e-09   1.10908e-06 DIIS
   @DF-UKS iter   6:  -410.24642384499242   -1.42575e-09   1.69157e-07 DIIS
   @DF-UKS iter   7:  -410.24642384502681   -3.43903e-11   2.58215e-08 DIIS
   @DF-UKS iter   8:  -410.24642384502800   -1.19371e-12   2.17169e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

   Electrons on quadrature grid:
      Nalpha   =   22.9999983568 ; deviation = -1.643e-06
      Nbeta    =   22.9999983568 ; deviation = -1.643e-06
      Ntotal   =   45.9999967136 ; deviation = -3.286e-06 

   @Spin Contamination Metric:  -4.618527782E-14
   @S^2 Expected:                0.000000000E+00
   @S^2 Observed:               -4.618527782E-14
   @S   Expected:                0.000000000E+00
   @S   Observed:                0.000000000E+00

    Orbital Energies [Eh]
    ---------------------

    Alpha Occupied:                                                       

       1B1g  -19.267516     1B2u  -19.267508 


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:22 2022
Module time:
	user time   =       6.83 seconds =       0.11 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          7 seconds =       0.12 minutes
Total time:
	user time   =      39.83 seconds =       0.66 minutes
	system time =       0.86 seconds =       0.01 minutes
	total time  =         41 seconds =       0.68 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:22 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: d2h
    Full point group: D2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  Reading orbitals from file /tmp/stdout.default.554391.180.npy, no projection.

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 0.928 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:               950
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0210
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Cached 100.0% of DFT collocation blocks in 0.469 [GiB].

  Minimum eigenvalue in the overlap matrix is 2.0567390718E-04.
  Reciprocal condition number of the overlap matrix is 3.070831

 Guess: Orbitals guess was supplied from a previous computation.

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     Ag        27      27       6       6       6       0
     B1g       21      21       4       4       4       0
     B2g       12      12       1       1       1       0
     B3g        9       9       1       1       1       0
     Au         9       9       1       1       1       0
     B1u       12      12       1       1       1       0
     B2u       21      21       4       4       4       0
     B3u       27      27       5       5       5       0
   -------------------------------------------------------
    Total     138     138      23      23      23       0
   -------------------------------------------------------

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-UKS iter   0:  -


*** tstop() called on B450-AORUS-PRO-WIFI at Thu Jun  2 11:52:30 2022
Module time:
	user time   =       5.50 seconds =       0.09 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          6 seconds =       0.10 minutes
Total time:
	user time   =      47.66 seconds =       0.79 minutes
	system time =       0.89 seconds =       0.01 minutes
	total time  =         49 seconds =       0.82 minutes

*** tstart() called on B450-AORUS-PRO-WIFI
*** at Thu Jun  2 11:52:30 2022


         ------------------------------------------------------------
                                   SCF GRAD                          
                          Rob Parrish, Justin Turney,                
                       Andy Simmonett, and Alex Sokolov              
         ------------------------------------------------------------

  ==> Geometry <==

    Molecular point group: d2h
    Full point group: D2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       C

  Reading orbitals from file /tmp/stdout.default.554391.180.npy, no projection.

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.078 GiB; user supplied 0.928 GiB. Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:               950
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0210
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (AUG-CC-PVDZ AUX)
    Blend: AUG-CC-PVDZ-JKFIT
    Number of shells: 168
    Number of basis functions: 516
    Number of Cartesian functions: 606
    Spherical Harmonics?: true
    Max angular momentum: 3

  Cached 100.0% of DFT collocation blocks in 0.469 [GiB].

  Minimum eigenvalue in the overlap matrix is 2.0591313796E-04.
  Reciprocal condition number of the overlap matrix is 3.074718

 Guess: Orbitals guess was supplied from a previous computation.

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     Ag        27      27       6       6       6       0
     B1g       21      21       4       4       4       0
     B2g       12      12       1       1       1       0
     B3g        9       9       1       1       1       0
     Au         9       9       1       1       1       0
     B1u       12      12       1       1       1       0
     B2u       21      21       4       4       4       0
     B3u       27      27       5       5       5       0
   -------------------------------------------------------
    Total     138     138      23      23      23       0
   -------------------------------------------------------

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-UKS iter   0:  -

KeyboardInterrupt: 

**Pregunta 4.** Calcule la energía de la molécula de $NO_2$ con DFT B3LYP y la base aug-cc-pvdz.

In [5]:
NO2 = psi4.geometry("""
0 2
N      0.0000  0.0000 0.0000 
O      0.0000  1.0989 0.4653 
O      0.0000 -1.0989 0.4653 
units angstrom
""")
psi4.set_options({'reference': 'uhf'})
no2=psi4.optimize("B3LYP/aug-cc-pvdz")
print(no2)

Optimizer: Optimization complete!
-205.1126037341589


**Pregunta b.** Calcule el $\Delta U$ de la reacción $N_2O_4 <=> 2NO_2$ según DFT B3LYP.

In [6]:
(2*no2-n2o4)*2625.5

55.71947103963211

## Referencias

- Parrish, R. M.; Burns, L. A.; Smith, D. G. A.; Simmonett, A. C.; DePrince, A. E.; Hohenstein, E. G.; Bozkaya, U.; Sokolov, A. Y.; Di Remigio, R.; Richard, R. M.; et al. **Psi4 1.1: An Open-Source Electronic Structure Program Emphasizing Automation, Advanced Libraries, and Interoperability.** Journal of Chemical Theory and Computation 2017, 13 (7), 3185–3197.
